tensorflow/examples/android/src/org/tensorflow/demo/StylizeActivity.java

```diff
@@ -381,6 +390,17 @@ public void drawCallback(final Canvas canvas) {
     grid = (GridView) findViewById(R.id.grid_layout);
     grid.setAdapter(adapter);
     grid.setOnTouchListener(gridTouchAdapter);
+
+    // Change UI on Android TV
+    UiModeManager uiModeManager = (UiModeManager) getSystemService(UI_MODE_SERVICE);
+    if (uiModeManager.getCurrentModeType() == Configuration.UI_MODE_TYPE_TELEVISION) {
+      RelativeLayout.LayoutParams layoutParams = new RelativeLayout.LayoutParams(400, ViewGroup.LayoutParams.MATCH_PARENT);
```

Comment:
Where does the 400 come from? Can it be computed programmatically?

Reply:
I am not sure how the textureView adjust its width in runtime.
Can you point me where I can read the width?